cmd/porter/outputs.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"github.com/deislabs/porter/pkg/porter"
+	"github.com/spf13/cobra"
+)
+
+func buildOutputsCommand(p *porter.Porter) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "outputs",
```

Comment:
I think that this would be better not as a standalone command but as a task-based command that is used to inspect an installed bundle.

So instead of having `porter outputs -b bundle name` (which is off anyway because we are actually working with claim names right?), what if we had a command that let you look at an installed bundle and also had it dig deeper into additional available into (like outputs) if the bundle had it?

Maybe something like this:

```console
$ porter bundles list
NAME                  CREATED      MODIFIED     LAST ACTION   LAST STATUS
mysql                  2019-06-24   2019-06-24   install       success

$ porter bundle show mysql
Name          mysql
Created       2019-06-24
Modified      2019-06-24
Last Action  install
Last Status  success

Outputs
=====================================================================
Name           Type         Value
-----------+--------+--------------------------------------------------------+
hostName     String          wpmysql
port                Integer       5423
clientCert       String         /Users/carolynvs/.porter/outputs/.../mysql/clientCert
```

clientCert is writeOnly so we don't display it. Find some logic for not dumping sensitive or giant info

Reply:
Roger; I can switch tactics and implement `porter bundle show [NAME]` to show outputs for a bundle.

Should we leave this PR with that command or do we also want a solution/command for getting a particular output value?  (Perhaps to merely echo/print or perhaps to send to a file, etc.)

Reply:
If someone wants to just get a single output value, I would encourage them to use the --output json and then use jq or something to grab what they need. Maybe we can start with that for now and see if people want a more specialize command like `porter bundle output list/show` to work directly with outputs. We may never need to implement it, depending on lack of asks/feedback.

How does that sound?

Reply:
That sounds good to me.  My only question revolves around the case when an actual output value is large (think: kubeconfig file) while listing.  Shall we just default to truncating all values at a certain character limit when printing via default/table and print the entire value if the printing option is otherwise (yaml or json)?

Reply:
Let's start with truncating at certain number of characters and see how well that works.

Reply:
Ok, updated to limit cli scope to just the `porter bundle show` command.  Please take a look when convenient.